### ISSUE TITLE
ProgrammingError at / - relation "cart_cart" does not exist

### ISSUE THREAD

**Issue by devax** [mirumee/saleor#1185]
Hi folks,
I just started today with checking out this amazing project! I hope my question/problem is a legit one ...
I have cloned the repository. I changed the Dockerfile to include a line: `ENV ALLOWED_HOSTS 192.168.33.5` to be able to access the page using the host's internal IP. Then I started the containers with `docker-compose up -d`. All containers seem to run as intended.

Anyways, when I try to connect hat [http://192.168.33.5:8000](url), I get this error: 

> ProgrammingError at / - relation "cart_cart" does not exist
(full error message in pdf below)
[ProgrammingError at _.pdf](https://github.com/mirumee/saleor/files/1382239/ProgrammingError.at._.pdf)

Am I missing something?

**Comment by elwoodxblues** [comment]
Have you run `python manage.py migrate`? It looks like database tables were not created at all.

**Comment by bogdal** [comment]
@devax Preparing the database should solve your problem:
```
docker-compose run web python manage.py migrate
```

**Comment by devax** [comment]
Thank you guys - sorry, I missed that one! I finally got it working. I'm leaving some additional information here, in case it will help someone else in future.

Next error was "OSError at / ...". I finally found the relevant installation manual: [https://saleor.readthedocs.io/en/latest/development.html#docker-dev](url).
It took me some time to figure out that `yarn` must be run on the host, not inside the container.
On the Synology NAS, these are the steps that worked for me:

1. From package manager, install Node.js 6 (beta)
2. In terminal, install yarn in the version needed: `sudo npm install --global yarn@1.0.2`
3. Even after re-login, `yarn` was not a recognized command in my case. So I used the output from nmp install command to point to yarn binary directly, like so: `sudo /volume1/@appstore/Node.js_v6/usr/local/bin/yarn`
4. `sudo /volume1/@appstore/Node.js_v6/usr/local/bin/yarn run build-assets`
Then proceed with the next steps from installation manual (`docker-compose build` etc.)
This made my installation on my Synology NAS work.
